tooling/nargo_cli/tests/snapshots/execute__tests__strings_force_brillig_true_inliner_0.snap

```diff
@@ -0,0 +1,79 @@
+---
```

Comment:
Is this snapshot okay? Asking because there are no new tests in this PR and this is a new snapshot.

Reply:
It looks like it got added from a transient test config because I deleted the file and tests are passing locally